src/Castle.Services.Logging.NLogIntegration/Castle.Services.Logging.NLogIntegration-VS2017.csproj

```diff
@@ -22,7 +22,7 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="NLog" Version="4.4.5" />
+    <PackageReference Include="NLog" Version="4.4.0-beta13" />
```

Comment:
Why are we going backwards to `4.4.0-beta13`, that one isn't even listed on nuget.org?

Ah, looks like that is what is in master, probably best for us to pick `4.4.0` for the next release then.